include/ProjectInfo.hh

```diff
@@ -0,0 +1,9 @@
+// other useful GEANT4 exported variables:
```

Comment:
This file should be removed, it has been replaced by the (git-excluded) `RMGConfig.hh`